packages/storybook/.storybook/preview-head.html

```diff
@@ -0,0 +1,11 @@
+<style>
+  body, html {
+    margin: 0;
```

Comment:
I wonder if our layout components should normalize via https://github.com/sindresorhus/modern-normalize or somesuch.

Reply:
Later, #15